website/docs/d/eventhub_consumer_group.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+subcategory: "Messaging"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_eventhub_consumer_group"
+description: |-
+  Gets information about an Event Hubs Consumer Group within an Event Hub.
+---
+
+# Data Source: azurerm_eventhub_consumer_group
+
+Use this data source to access information about an existing Event Hubs Consumer Group within an Event Hub.
+
+## Example Usage
+
+```hcl
+data "azurerm_eventhub_consumer_group" "test" {
+  name                = "${azurerm_eventhub_consumer_group.test.name}"
+  namespace_name      = "${azurerm_eventhub_namespace.test.name}"
+  eventhub_name       = "${azurerm_eventhub.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the EventHub Consumer Group resource.
+
+* `namespace_name` - (Required) Specifies the name of the grandparent EventHub Namespace.
+
+* `eventhub_name` - (Required) Specifies the name of the EventHub.
+
+* `resource_group_name` - (Required) The name of the resource group in which the EventHub Consumer Group's grandparent Namespace exists.
```

Comment:
we don't need required/optional on data source documentation, and could we use language consistent with the rest of the provider?